tests/tools/src/cuda/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+#Detray library, part of the ACTS project(R& D line)
+#
+#(c) 2024 CERN for the benefit of the ACTS project
+#
+#Mozilla Public License Version 2.0
+
+# C++17 support for CUDA requires CMake 3.18.
+cmake_minimum_required(VERSION 3.18)
+
+# Enable CUDA as a language.
+enable_language(CUDA)
+
+# Set the CUDA build flags.
+include(detray-compiler-options-cuda)
+
+include( CMakeFindDependencyMacro )
+
+find_dependency( Boost COMPONENTS program_options )
```

Please don't use `find_dependency` outside of `detray-config.cmake`. That macro is meant **only** for files that would describe an installed project.

Just use:

```suggestion
find_package( Boost COMPONENTS program_options REQUIRED )
```

---

Done